presto-main/src/main/java/com/facebook/presto/operator/AnalyzeFinishOperator.java

```diff
@@ -119,7 +128,30 @@ public Page getOutput()
 
         Collection<ComputedStatistics> computedStatistics = computedStatisticsBuilder.build();
         analyzeFinisher.finishAnalyze(computedStatistics);
-        return null;
+
+        // output page will only be constructed once,
+        // so a new PageBuilder is constructed (instead of using PageBuilder.reset)
+        PageBuilder page = new PageBuilder(1, TYPES);
+        page.declarePosition();
+
+        boolean hasRowCount = false;
```

Comment:
Extract this to a method

`private static OptionalLong getRowsCount(Collection<ComputedStatistics> computedStatistics)`